apps/chatbot/docker/files/.aws/credentials

```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id = 123
+aws_secret_access_key = xyz
```

Comment:
If the docker container needs to call actual AWS resources (for example bedrock), I think this may result in unauthorized access.
I've commonly seen using the option C from [this answer](https://stackoverflow.com/a/56077990). This way, the container uses your AWS credentials and configuration

Reply:
I tested and it works because our code uses `CHB_AWS_*` .
The problem is the local dynamodb that needs the dummy profile.

Reply:
Doesn't it work with real credentials?